src/main/scala/dsptools/VerboseDspTesterUtils.scala

```diff
@@ -0,0 +1,135 @@
+package dsptools
+
+import chisel3._
+import java.io.{File, FileWriter, BufferedWriter}
+import chisel3.iotesters.TestersCompatibility
+
+// Note: This will dump as long as genVerilogTb is true (even if you're peeking/poking DspReal)
+trait VerilogTbDump {
```

Comment:
Mixing traits like that doesn't seem to work. I can't access stuff like incTime, etc. from the trait.